### PR TITLE
Fix failing tests in master

### PR DESCRIPTION
**Pull request by kenfinnigan**
I know the fix isn't ideal, as it adds sleep, but any alternative suggestions are appreciated

### CODE REVIEW

**Review by geoand — APPROVED** [review]
Let's get this in sooner rather than later

File: extensions/resteasy/deployment/src/test/java/io/quarkus/resteasy/test/NotFoundExceptionMapperTestCase.java [hunk 1]
```diff
@@ -54,15 +54,19 @@ public void testJsonResourceNotFound() {
     }
 
     @Test
-    public void shouldDisplayNewAddedFileIn404ErrorPage() {
+    public void shouldDisplayNewAddedFileIn404ErrorPage() throws InterruptedException {
         String CONTENT = "html content";
         test.addResourceFile(META_INF_RESOURCES + "index2.html", CONTENT);
 
+        Thread.sleep(700);
```
Comment (cescoffier) [inline extensions/resteasy/deployment/src/test/java/io/quarkus/resteasy/test/NotFoundExceptionMapperTestCase.java]: wouldn't it be better to use Awaitibility and poll until it passes?

Comment (kenfinnigan) [reply extensions/resteasy/deployment/src/test/java/io/quarkus/resteasy/test/NotFoundExceptionMapperTestCase.java]: Await on the `RestAssured` call passing?

And then have a max time of 2 seconds?

Is that preferred? I can adjust

Comment (kenfinnigan) [reply extensions/resteasy/deployment/src/test/java/io/quarkus/resteasy/test/NotFoundExceptionMapperTestCase.java]: We currently don't have Awaitility as a dependency, is it ok to add?

Comment (cescoffier) [reply extensions/resteasy/deployment/src/test/java/io/quarkus/resteasy/test/NotFoundExceptionMapperTestCase.java]: I have extensions using it I believe.

Comment (kenfinnigan) [reply extensions/resteasy/deployment/src/test/java/io/quarkus/resteasy/test/NotFoundExceptionMapperTestCase.java]: ugh, I was looking at the wrong BOM!

Comment (kenfinnigan) [reply extensions/resteasy/deployment/src/test/java/io/quarkus/resteasy/test/NotFoundExceptionMapperTestCase.java]: Tried switching to awaitility, and it has a failure with one, not entirely sure why

Comment (kenfinnigan) [reply extensions/resteasy/deployment/src/test/java/io/quarkus/resteasy/test/NotFoundExceptionMapperTestCase.java]: Awaitility seems to lead to just as much raciness unfortunately

**Pull request merged**